### PR TITLE
edgex-go swagger fixes

### PR DESCRIPTION
**Pull request by ernestojeda**
created new script to wrap multiple call to publish different API versions.
This will make it easier to migrate this script to pipelines in the future.

Signed-off-by: Ernesto Ojeda <ernesto.ojeda@intel.com>

### CODE REVIEW

**Comment by ernestojeda** [comment]
recheck

**Comment by ernestojeda** [comment]
Build failed due to python error. Trying recheck

**Review by soda480 — APPROVED** [review]
Looks good - I like the DryRun capability. Only question I have is why the OWNER and OASVERSION are hard coded as opposed to derived from the environment.

File: shell/toSwaggerHub.sh [hunk 1]
```diff
@@ -1,17 +1,41 @@
...
+                    -H "Content-Type:application/yaml" \
+                    -d "${apiContent}"
+            else
+                echo "[toSwaggerHub] Dry Run enabled...Simulating upload"
+                echo "curl -X POST https://api.swaggerhub.com/apis/${owner}/${apiName}?isPrivate=${isPrivate}&version=${apiVersion}&oas=${oasVersion}&force=true"
+            fi
+
+        done
+    else
+        echo "Could not API Folder [${apiPath}]. Please make sure the API version exists..."
```
Comment (jamesrgregg) [inline shell/toSwaggerHub.sh]: are we missing the word "find" in this echo statement?

**Review by jamesrgregg — APPROVED** [review]
Other than the missing word in the echo statement.. LGTM

File: shell/toSwaggerHub.sh (hunk 1 above)
Comment (ernestojeda) [reply shell/toSwaggerHub.sh]: sure. I will add it.

**Comment by ernestojeda** [comment]
recheck

**Pull request merged**